app/services/fact_parser.rb

```diff
@@ -1,6 +1,6 @@
 class FactParser
   delegate :logger, :to => :Rails
-  VIRTUAL = /\A([a-z0-9]+)_([a-z0-9]+)\Z/
+  VIRTUAL = /\A([a-z0-9]+)[_\.]+([a-z0-9]+)\Z/
```

Comment:
The plus character is I believe not needed, do you have some real example of having `eth0__something`? Is this some practice you have this for bridges? If so, let's just cover this in a test below.

Reply:
@lzap no problem to collaborate, you always can reach me by mail or telegram (for example). There is no need in plus sign, it added just in case. This regexp well tested as long as the whole patchset used in my production.